src/extra/libs/img_font/lv_img_font.c

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file lv_img_font.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_img_font.h"
+
+#if LV_USE_IMG_FONT
+
+/*********************
+ *      DEFINES
+ *********************/
+#define LV_IMG_FONT_PATH_MAX_LEN 64
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+typedef struct {
+    lv_img_font_t * info;
+    char path[LV_IMG_FONT_PATH_MAX_LEN];
+} img_dsc_t;
```

Comment:
Wouldn't be clearer to add an `lv_font_t` field here directly instead of in in `info`?